Emby.Server.Implementations/Updates/InstallationManager.cs

```diff
@@ -47,49 +48,15 @@ public class InstallationManager : IInstallationManager
         /// </summary>
         public event EventHandler<GenericEventArgs<IPlugin>> PluginUninstalled;
 
-        /// <summary>
-        /// Called when [plugin uninstalled].
-        /// </summary>
-        /// <param name="plugin">The plugin.</param>
-        private void OnPluginUninstalled(IPlugin plugin)
-        {
-            PluginUninstalled?.Invoke(this, new GenericEventArgs<IPlugin> { Argument = plugin });
-        }
```

Comment:
I don't see a replacement for this event anywhere?

Reply:
https://github.com/jellyfin/jellyfin/pull/1462/files/65a0ca2f32e2eba640fbfead46ffb5bfd02e2c88#diff-b38c8fe5bc172bada6c68b600f7cd290R574
It got inlined